Concepts/Tic-Tac-Toe/Services/GameManager.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Tic_Tac_Toe.Entities;
+using Tic_Tac_Toe.Services.Interfaces;
+
+namespace Tic_Tac_Toe.Services
```

Comment:
Tic_Tac_Toe => follow the naming convention.

Reply:
Yes changed it